.github/workflows/label-templated-discussions.yml

```diff
@@ -0,0 +1,97 @@
+name: Label Templated Discussions
+
+on:
+  discussion:
+    types: [created]
+
+jobs:
+  label-templated-discussion:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get selected topic
+        id: get_selected_topic
+        uses: actions/github-script@v6
+        env:
+          DISCUSSION_HTML: ${{ github.event.discussion.html }}
```

Comment:
Needs an additional step to fetch the discussion body - it's not included in the payload.